applications/omnipresence/src/omnip_message_summary.erl

```diff
@@ -229,43 +220,7 @@ send_update(<<"amqp">>, _User, Props, Subscriptions) ->
                             )
          || S <- Stalkers
         ],
-    wh_amqp_worker:checkin_worker(Worker);
-send_update(<<"sip">>, User, Props, Subscriptions) ->
```

Comment:
@lazedo 
Why are you removing this here and above?  I think this is possibly useful for future deployments, and it works no?  Why not leave it?